content/en/docs/components/katib/getting-started.md

```diff
@@ -17,6 +17,31 @@ You need to install the following Katib components to run examples:
 
 ## Getting Started with Katib Python SDK
 
+### Adding Namespace Label for the Metrics Collector Injection
+
+Before running your hyperparameter tuning Katib Experiment with Python SDK, 
+ensure the namespace label `katib.kubeflow.org/metrics-collector-injection: enabled` 
+is present. This label enables the sidecar container injection to collect metrics during the experiment.
+
+You can configure the namespace using the following YAML:
```

Comment:
Since the user needs to update the namespace, no create a new one. I'd update the sentence by
"You can configure the namespace by adding the following label  katib.kubeflow.org/metrics-collector-injection: enabled as is shown in the sample code."